### ISSUE TITLE
Valid code with count.index marked as issue

### ISSUE THREAD

**Issue by N0zz** [terraform-linters/tflint#448]
While creating aws_route53_health_checks while using count option I get some data as output from this resource. 

Next step is to access this data with aws_route53_record module and create some records associated with healthchecks that were created. When I try to do this this way:

```
health_check_id  = "${aws_route53_health_check.aws_health_check[count.index].id}"
```
tflint (used in vscode) marks this as an issue that needs to be fixed. However this is valid terraform code that works on prod already.
Error:
```
expected "}" but found "."
```

I realise that this might not be best way to create healthchecks and records associated with those healthchecks. But it'll take some time before we start to refactor this code. And until then, my whole repository is glowing red, because tflint decided that this code is invalid.

**Comment by wata727** [comment]
Hi @N0zz 

Could you share the results of running TFLint directly without VS Code? As far as I know, this is probably not a TFLint issue.

Duplicate of #330

**Comment by wata727** [comment]
Close this due to no activity.
